modules/openshift-lightspeed-web-console.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * capabilities-web-console.adc
+
+:_mod-docs-content-type: CONCEPT
+[id="openshift-lightspeed-web-console_{context}"]
+=  Enhancing your experiance in the {product-title} web console by using {ols-official}
```

Comment:
```suggestion
=  Enhancing your experience in the {product-title} web console by using {ols-official}
```

Reply:
Maybe the title could be simplified as "Red Hat OpenShift Lightspeed in the web console", to keep consistent with other following capabilities, such as:
"Red Hat OpenShift Pipelines in the web console"
@opayne1 WDYT?